frontend-react/src/components/EditableCompare.tsx

```diff
@@ -66,43 +67,12 @@ export const EditableCompare = forwardRef(
         // the API call into this forwardRef well say if json is valid (to enable save button)
         const [isValidSyntax, setIsValidSyntax] = useState(true);
 
-        useImperativeHandle(
-            ref,
-            () => ({
-                getEditedText() {
-                    return textAreaContent;
-                },
-                getOriginalText() {
-                    return props.original;
-                },
-                // when showing/hiding json, force am update of the content
-                refreshEditedText(updatedjson) {
-                    setTextAreaContent(updatedjson);
-                    onChangeHandler(updatedjson);
-                },
-                isValidSyntax() {
-                    return isValidSyntax;
-                },
-            }),
-            // onChangeHandler appears below, remove from deps
-            // eslint-disable-next-line
-            [textAreaContent]
-        );
-
-        const turnOffSpellCheckSwigglies = () => {
```

Comment:
I'm not sure why we needed a function for this since there's a [`spellcheck` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck) (`spellCheck` as a React prop).  I've updated the textarea [below](https://github.com/CDCgov/prime-reportstream/pull/7713/files#diff-9549b9587cf857fa88a11d9c5d5a2ad10d106f0846b78007cd5901d000c038b8R228), but let me know if there's was a reason for this function!

Reply:
I vaguely remember struggling to get the HTML attribute approach to work, but maybe that issue long resolved?

See: https://stackoverflow.com/questions/49079148/reactjs-remove-spellcheck-attribute